app/models/product.rb

```diff
@@ -39,7 +39,7 @@ def self.categories
   end
 
   def average_rating
-    return "Nothing to show" if reviews.count == 0
+    return "Be the first to review this item" if reviews.count == 0
```

Comment:
Maybe we should return something else since we already say "Be the first to review this item" below. Makes the page look a bit repetitive.